tests/unit/test_orchestrator.py

```diff
@@ -198,5 +198,5 @@ def test_extend_parser_zuul_system(self):
         self.assertTrue("jobs" in self.orchestrator.parser.ci_args)
         self.assertTrue("builds" in self.orchestrator.parser.ci_args)
         self.assertEqual(self.orchestrator.parser.ci_args["tenants"].level, 2)
```

Comment:
Maybe it would be a good idea to add in this tests the new models (pipelines, projects) and explicitly check their level

Reply:
Done.